packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.styles.ts

```diff
@@ -0,0 +1,303 @@
+import { IStyle, HighContrastSelector, ScreenWidthMaxSmall, getScreenSelector, getFocusStyle } from '../../Styling';
+import { IMessageBarStyleProps, IMessageBarStyles, MessageBarType } from './MessageBar.types';
+import { IsFocusVisibleClassName } from '@uifabric/utilities';
+
+export const getStyles = (props: IMessageBarStyleProps): IMessageBarStyles => {
+  const { theme, className, messageBarType, onDismiss, actions, truncated, isMultiline, expandSingleLine } = props;
+  const { semanticColors, palette, fonts } = theme;
+
+  const SmallScreenSelector = getScreenSelector(0, ScreenWidthMaxSmall);
+
+  const errorStyle = {
+    background: semanticColors.errorBackground,
+    selectors: {
+      '& .icon': {
+        color: semanticColors.errorText
+      }
+    }
+  };
+
+  const blockedStyle = {
+    background: semanticColors.errorBackground,
+    selectors: {
+      '& .icon': {
+        color: semanticColors.errorText
+      }
+    }
+  };
+
+  const severeWarningStyle = {
+    background: semanticColors.blockingBackground,
+    selectors: {
+      '& .icon': {
+        color: semanticColors.errorText
+      }
+    }
+  };
+
+  const successStyle = {
+    background: semanticColors.successBackground,
+    selectors: {
+      '& .icon': {
+        color: palette.green
+      }
+    }
+  };
+
+  const warningStyle = {
+    background: semanticColors.warningBackground
+  };
+
+  const dismissalAndExpandStyle: IStyle = {
+    flexShrink: 0,
+    margin: 8,
+    marginLeft: 0,
+    selectors: {
+      '& .ms-Button-icon': {
+        fontSize: 12,
+        height: 12,
+        lineHeight: 12,
```

Comment:
please attach units for line-height
CSS defaults to EMs, not pixels, not sure what we do but we were discussing changing it